### ISSUE TITLE
Inconsintent boilerplate at top of files

### ISSUE THREAD

**Issue by petervanderdoes** [wger-project/wger#392]
The license at the top of each `*.py` is inconsistent.

- Some don't have the license boilerplate `wger/email/migrations/0001_initial.py`
- Some don't have the line `# along with Workout Manager.  If not, see <http://www.gnu.org/licenses/>.` in the license part
- Some don't have the correct file encoding line `# -*- coding: utf-8 *-*`

Anyway as you can see it's inconsistent throughout.

#### Suggestions
- Fix the encoding line for all `*.py` files
- Have a consistent license boilerplate throughout. Unless you want to get rid of the boilerplate which is ok with me as well.

#### Suggested License Boiler Plate
Use the boilerplate as suggested by the AGPL license itself

    wger - A web application to manage your exercises and personal workouts, weight and diet plans.
    Copyright (C) 2017 Roland Geider

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.

    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.

**Comment by rolandgeider** [comment]
Yeah, there is no boilerplate at all in generated files such as the migrations, which is probably fine. As for making the header more consistent as well as the encoding (is this still important for python 3?), I'm all for it. We can use the text you suggested, I'm not sure where I got the one I used, I could have sworn it was from the gnu.org site as well.
